Makefile

```diff
@@ -571,3 +571,9 @@ mv $(1) $(1)-$(3) ;\
 } ;\
 ln -sf $(1)-$(3) $(1)
 endef
+
+
+.PHONY: defaults
+defaults:
+	@echo "generating defaults"
+	hack/generate-defaults.sh
```

Comment:
which files this script will generate?

Reply:
everything under the defaulter package but the `defaulter.go` file.

I think I'll move all the generated code into a separate dir inside `defaulter`